README.md

```diff
@@ -23,3 +23,22 @@ All Homeworks branches related to this repo are and will be merge with the devel
 ### *- Nice to have (for Above and Beyond):*
 - [x] Use default values with functions
 - [x] Use maps to store hardcoded data instead of ArrayList
+
+
+## Week 3 Homework
+
+### *- Need to have (for passing grade):*
+- [x] Switch following Source Properties to use enum classes:
+  #### . Category
+   #### . Language
+    #### . Country    
+- [x] Create a  **NewsService** with a single method to get articles.
+- [x] Reference **NewsService** from Activity and get the articles.
```

Comment:
I couldn't find where the NewsService was referenced in MainActivity, instead it has it's own list of articles.